pkg/webhook/work/mutating.go

```diff
@@ -64,6 +66,8 @@ func (a *MutatingAdmission) Handle(_ context.Context, req admission.Request) adm
 			return admission.Errored(http.StatusInternalServerError, err)
 		}
 
+		setAssociatedLabelsAndAnnotations(workloadObj, work)
```

Comment:
Should we skip this operation for resources collected from member clusters, such as EndpointSlice?

Reply:
why it should be skipped?

Reply:
Do they use these labels and annotations?

Reply:
maybe not. but the behavior now is almost the same as before, and we can treat it as standard behavior, I think it's ok.

Reply:
I think we can distinguish between downward synchronization and upward collection. These are two modes. It's better to wait until the need arises in the future.

Reply:
Currently, we use the label `propagation.karmada.io/instruction` to identify the EndpointSlice collected from the cluster.

Reply:
thanks